data/database.py

```diff
@@ -755,6 +756,26 @@ class RobotAccountToken(BaseModel):
     fully_migrated = BooleanField(default=False)
 
 
+class QuotaLimitGroups(BaseModel):
+    group_name = CharField()
+
+
+class QuotaType(BaseModel):
+    description = CharField()
+
+
+class QuotaLimits(BaseModel):
+    quota_limit_group_id = ForeignKeyField(QuotaLimitGroups)
+    quota_type_id = ForeignKeyField(QuotaType)
```

Comment:
There's a wrapper for `ForeignKeyField`, `EnumField` that adds some nice-to-have methods to it,

Reply:
If there isnt any chance for unnecessary overhead we can swap them out but for my purposes this has everything I need.